backend/src/Notifo.Domain.Integrations/Smtp/SmtpIntegration.cs

```diff
@@ -42,7 +42,6 @@ public sealed partial class SmtpIntegration : IIntegration
 
     public static readonly IntegrationProperty FromEmailProperty = new IntegrationProperty("fromEmail", PropertyType.Text)
     {
-        Pattern = Patterns.Email,
```

Comment:
Can you put this property at the end, just like in the other property?. My brain is annoying sometimes ;)